files/whitestormjs/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "whitestormjs",
+  "repo": "sasha240100/WhitestormJS",
+  "files": {
+    "include": ["build/whitestorm.min.js", "build/whitestorm.js"]
```

Comment:
There is no `build` directory here https://github.com/sasha240100/WhitestormJS/tree/v0.1-beta.3
The bot will fail